.github/workflows/CI.yml

```diff
@@ -157,32 +156,15 @@ jobs:
     - uses: actions/setup-python@v2
       with:
         python-version: 3.6
-    - name: Install Rust toolchain
-      uses: actions-rs/toolchain@v1
-      with:
-        toolchain: stable
-        target: ${{ matrix.platform.target }}
-        profile: minimal
-        default: true
-    - name: Install aarch64 cross compiler
-      if: matrix.platform.target == 'aarch64-unknown-linux-gnu'
-      run: |
-        sudo apt-get install -y gcc-aarch64-linux-gnu libc6-arm64-cross libc6-dev-arm64-cross
-        echo "TARGET_CC=aarch64-linux-gnu-gcc" >> "$GITHUB_ENV"
-        echo "TARGET_CXX=aarch64-linux-gnu-cpp" >> "$GITHUB_ENV"
-        echo "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc" >> "$GITHUB_ENV"
-    - name: Install armv7 cross compiler
-      if: matrix.platform.target == 'armv7-unknown-linux-gnueabihf'
-      run: |
-        sudo apt-get install -y gcc-arm-linux-gnueabihf libc6-armhf-cross libc6-dev-armhf-cross
-        echo "TARGET_CC=arm-linux-gnueabihf-gcc" >> "$GITHUB_ENV"
-        echo "TARGET_CXX=arm-linux-gnueabihf-cpp" >> "$GITHUB_ENV"
-        echo "CARGO_TARGET_ARMV7_UNKNOWN_LINUX_GNUEABIHF_LINKER=arm-linux-gnueabihf-gcc" >> "$GITHUB_ENV"
-    - name: Install maturin
-      run: pip install maturin
     - name: Build Wheels
       run: |
-        maturin build -i python --release --out dist --no-sdist --target ${{ matrix.platform.target }} --manylinux ${{ matrix.platform.manylinux }} --cargo-extra-args="--no-default-features" --cargo-extra-args="--features=abi3"
+        echo 'curl -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain stable
+        source ~/.cargo/env
+        rustup target add ${{ matrix.platform.target }}
+        maturin build -i python --release --out dist --no-sdist --target ${{ matrix.platform.target }} --manylinux ${{ matrix.platform.manylinux }} --cargo-extra-args="--no-default-features" --cargo-extra-args="--features=abi3,extension-module"  # disable mimallocator
+        ' > build-wheel.sh
+
+        docker run --rm -v "$PWD":/io -w /io messense/manylinux2014-cross:${{ matrix.platform.arch }} bash build-wheel.sh
```

Comment:
Using ubuntu-16.04 causing `cannot allocate memory in static TLS block` error which I believe is a issue with glibc. [messense/manylinux2014-cross](https://github.com/messense/manylinux2014-cross-arm) docker images use glibc 2.17 which is compliant with manylinux2014 on aarch64 and armv7l.

https://github.com/messense/pyrus-cramjam/runs/2101449199?check_suite_focus=true